### ISSUE TITLE
Game Instructions

### ISSUE THREAD

**Issue by smithad15** [rangle/block-drop#13]
The player should be able to access instructions for the game rules and controls from the game screen.

**Comment by bennett000** [comment]
In terms of where we'll store the text for the game instructions I would like to put it into the game config where the rules will live after #57 - this might need to be its own card but I wanted to track the notion somewhere :)
